example/yarn.lock

```diff
@@ -11649,8 +11649,8 @@ worker-rpc@^0.1.0:
   dependencies:
     microevent.ts "~0.1.1"
 
-world_countries_lists@^2.1.0:
-  version "2.1.0"
+world_countries_lists@^2.4.0:
+  version "2.4.0"
   resolved "https://registry.yarnpkg.com/world_countries_lists/-/world_countries_lists-2.1.0.tgz#f93447bc7e40955fc9edde99351ece4ab67e4cda"
```

Comment:
I'm afraid this package is not updated. Try to run `yarn add world_countries_lists` **under each demo project directory**.